src/components/NavBar/ShoppingBag.tsx

```diff
@@ -2,14 +2,33 @@ import { NavIcon } from 'components/NavBar/NavIcon'
 import * as styles from 'components/NavBar/ShoppingBag.css'
 import { Box } from 'nft/components/Box'
 import { BagIcon, HundredsOverflowIcon } from 'nft/components/icons'
-import { useBag } from 'nft/hooks'
-import { useEffect, useState } from 'react'
+import { useBag, useSellAsset } from 'nft/hooks'
+import { useCallback, useEffect, useState } from 'react'
+import shallow from 'zustand/shallow'
 
 export const ShoppingBag = () => {
   const itemsInBag = useBag((state) => state.itemsInBag)
   const [bagQuantity, setBagQuantity] = useState(0)
 
-  const toggleBag = useBag((s) => s.toggleBag)
+  const { bagExpanded, setBagExpanded } = useBag(
+    ({ bagExpanded, setBagExpanded }) => ({ bagExpanded, setBagExpanded }),
+    shallow
+  )
+  const { isSellMode, resetSellAssets, setIsSellMode } = useSellAsset(
```

Comment:
same here, what is this shallow for?

Reply:
<img width="1036" alt="image" src="https://user-images.githubusercontent.com/5773490/200924932-c8f4b6e9-1ede-4404-ae17-46f61dd34fbc.png">
https://github.com/pmndrs/zustand/blob/d27ea948843b2c955ce2be530e5e963dff8958de/readme.md

shallow primitive value equality--won't work with deeply nested state, iirc